R/formula.R

```diff
@@ -30,15 +30,15 @@ formula_creator <- function(k, m, is_grouped) {
   if (m > 1) {
     # With adaptive basis, m refers to the order of the penalty matrix not the
     # order of the smoothing penalty as it does in the other smoothing bases.
-    plus_global_trend <- glue::glue("+ s(timesteps,
+    plus_global_trend <- glue::glue("+ s(timestep,
```

Comment:
Not directly relevant to this PR, but I think we should agree as a team on a string manipulation standard so that we are all using the same syntax and dependencies in our code.

Reply:
voting for `glue`

Reply:
+1